packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -385,34 +391,67 @@ export class DefaultCrudRepository<
     }
   }
 
+  /**
+   * Template method to amend filters with foreign key sources
```

Comment:
Could you add tests for this function and make the description more clear? for example what issues it tries to address, and maybe add an example like [this](https://github.com/strongloop/loopback-next/pull/5354/files#diff-4eac471411475eed05f0d6bcb873e607R29-R53)

Reply:
It's gone in the latest iteration